pyproject.toml

```diff
@@ -8,10 +8,20 @@ where = ["src"]
 [project]
 name = "schistosomiasis"
 version = "0.1.0"
-dependencies = ["pygeoprocessing", "taskgraph", "matplotlib", "natcap.invest @ git+https://github.com/natcap/invest.git@feature/plugins"]
+dependencies = [
+    "pygeoprocessing",
+    "taskgraph",
+    "matplotlib",
+    "natcap.invest @ git+https://github.com/natcap/invest.git@feature/plugins",
```

Comment:
Should this now point to the invest-schisto repo?

Reply:
Are there any differences in `natcap/schistosomiasis-invest` that affect the python package? If it's only the workbench that's different, I think this can stay the same. Eventually when `feature/plugins` is merged and released, this should just be `natcap.invest` and not be specific to the particular plugin.

Reply:
Gotchya, yea, I think it's just the Workbench side that's different.